### PR TITLE
Apples now taste like apples

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
add: Centcom now reports that thanks to extensive bioengineering, apples
and oranges now taste of apples and oranges, rather than nothing as they
did before.
/:cl:

Basically, my hack to give hydroponic grown things flavour when I
finished my taste PR never worked, and since I've started being a chef
of recent, this has been bugging me.

I think I changed some descriptions of some foods a bit.

### CODE REVIEW

File: code/modules/hydroponics/grown/beans.dm [hunk 1]
```diff
@@ -27,6 +27,7 @@
 	bitesize_mod = 2
 	foodtype = VEGETABLES
 	grind_results = list("soymilk" = 0)
+	tastes = list("soy" = 1)
```
Comment (Qustinnus) [inline code/modules/hydroponics/grown/beans.dm]: my favorite

File: code/modules/hydroponics/grown/cereals.dm [hunk 2]
```diff
@@ -87,6 +90,7 @@
 	seed = /obj/item/seeds/wheat/meat
 	foodtype = MEAT | GRAIN
 	grind_results = list("flour" = 0, "blood" = 0)
+	tastes = list("meatwheat" = 1)
```
Comment (Xhuis) [inline code/modules/hydroponics/grown/cereals.dm]: nitpick: maybe `list("meat" = 1, "wheat" = 1)` instead?

**Comment by Iamgoofball** [comment]
tesst

**Comment by Iamgoofball** [comment]
i cant apply a line comment for some reason but the poison berry description was a joke, changing the phrasing breaks the joke

File: code/modules/hydroponics/grown/beans.dm (hunk 1 above)
Comment (Anonmare) [reply code/modules/hydroponics/grown/beans.dm]: @Qustinnus Soyboy

File: code/modules/hydroponics/grown/mushrooms.dm [hunk 3]
```diff
@@ -277,6 +277,7 @@
 	icon_state = "glowcap"
 	filling_color = "#00FA9A"
 	effect_path = /obj/structure/glowshroom/glowcap
+	tastes = list("glowcap" = 1)
```
Comment (MoreRobustThanYou) [inline code/modules/hydroponics/grown/mushrooms.dm]: lamp, mushroom

File: code/modules/hydroponics/grown/berries.dm [hunk 4]
```diff
@@ -66,10 +68,11 @@
 /obj/item/reagent_containers/food/snacks/grown/berries/death
 	seed = /obj/item/seeds/berry/death
 	name = "bunch of death-berries"
-	desc = "Taste so good, you could die!"
+	desc = "Taste so good, you will die!"
```
Comment (MoreRobustThanYou) [inline code/modules/hydroponics/grown/berries.dm]: don't ruin the joke

File: code/modules/hydroponics/grown/berries.dm [hunk 5]
```diff
@@ -43,11 +44,12 @@
 /obj/item/reagent_containers/food/snacks/grown/berries/poison
 	seed = /obj/item/seeds/berry/poison
 	name = "bunch of poison-berries"
-	desc = "Taste so good, you could die!"
+	desc = "Taste so good, you might die!"
```
Comment (MoreRobustThanYou) [inline code/modules/hydroponics/grown/berries.dm]: don't ruin the joke

File: code/modules/hydroponics/grown/cereals.dm (hunk 2 above)
Comment (JJRcop) [reply code/modules/hydroponics/grown/cereals.dm]: second this

File: code/modules/hydroponics/grown/berries.dm (hunk 5 above)
Comment (AlexTheSysop42) [reply code/modules/hydroponics/grown/berries.dm]: >told to not ruin the joke
>changes it
>still ruins the joke

Comment (JJRcop) [reply code/modules/hydroponics/grown/berries.dm]: these are two different berries @Alek2ander

Comment (AlexTheSysop42) [reply code/modules/hydroponics/grown/berries.dm]: @JJRcop oh

also you pinged the wrong guy

Comment (JJRcop) [reply code/modules/hydroponics/grown/berries.dm]: maybe he'll come back!

File: code/modules/hydroponics/grown/berries.dm [hunk 6]
```diff
@@ -196,3 +203,4 @@
 	name = "bunch of green grapes"
 	icon_state = "greengrapes"
 	filling_color = "#7FFF00"
+	tastes = list("green grape" = 1)
```
Comment (chesse20) [inline code/modules/hydroponics/grown/berries.dm]: grapes dont taste diffrent based on what color they are as far as i know

Comment (Qustinnus) [reply code/modules/hydroponics/grown/berries.dm]: They do though

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-1). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**